transport/src/main/java/io/netty/channel/embedded/EmbeddedChannel.java

```diff
@@ -275,6 +281,13 @@ public boolean isActive() {
         return inboundMessages;
     }
 
+    public Queue<AbstractMap.SimpleEntry<Object, ChannelPromise>> tempInboundMessages() {
```

Comment:
Hi, @normanmaurer @njhill 
Since this method will be used in unit test, I make this public again..

Reply:
This is not needed in unit tests... please keep it private